demos/device_defender_for_aws/defender_config.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
This file belongs in each board's config_files folder

Reply:
Make sense. Will rename as "aws_defender_config.h" and move to Mediatek vendor demo config folder. For the other board that support lwip, such as ESP32, will add later when we verified the lwip metric on ESP32.

Reply:
I just needs to be called defender_config.h without the "aws_" prefix as that refers to the old library which we will not delete yet.

Reply:
got it.